meteor/client/ui/Shelf/ShowStylePanel.tsx

```diff
@@ -35,17 +35,21 @@ class ShowStylePanelInner extends MeteorReactComponent<Translated<IShowStylePane
 
 		return (
 			<div
-				className="show-style-panel timing"
+				className="show-style-panel"
 				style={isDashboardLayout ? dashboardElementStyle(this.props.panel as DashboardLayoutShowStyleDisplay) : {}}
 			>
-				<span className="timing-clock left">
-					<span className="timing-clock-label">{t('Show Style')}</span>
-					<span className="name">{this.props.showStyleBase.name}</span>
-				</span>
-				<span className="timing-clock left">
-					<span className="timing-clock-label">{t('Show Style Variant')}</span>
-					<span className="name">{this.props.showStyleVariant.name}</span>
-				</span>
+				<div className="show-style-subpanel">
+					<div className="show-style-subpanel__label">{t('Show Style')}</div>
+					<div className="show-style-subpanel__name" title={this.props.showStyleBase.name}>
```

Comment:
Is it really necessary to use div's here? Spans should do the job just fine
```
<div className="show-style-subpanel">
  <span className="show-style-subpanel__label">...</span>
  <span className="show-style-subpanel__name">...</span>
</div>
```

Some goes for the panel beneath this